### PR TITLE
Removed Table View

### PR DESCRIPTION
**Pull request by formalhaunt**
### Description of the Change

Removed table view as it has been superseded by table view 2.

### Why Should This Be In Core?

This is old code that is no longer being used.

### Benefits

It will remove dead code simplifying the code base.

### CODE REVIEW

File: CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/docs/tableview-toc.xml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE toc PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp TOC Version 2.0//EN" "http://java.sun.com/products/javahelp/toc_2_0.dtd">
-<toc version="2.0">
-    <tocitem text="Views">
-        <tocitem text="Table View" target="au.gov.asd.tac.constellation.views.tableview2.TableViewTopComponent"/>
-        <tocitem text="Table View (Experimental)" target="au.gov.asd.tac.constellation.views.tableview.about"/>
```
Comment (antares1470) [inline CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/docs/tableview-toc.xml:6]: similar

File: CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/docs/tableview-map.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE map PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 2.0//EN" "http://java.sun.com/products/javahelp/map_2_0.dtd">
-<map version="2.0">
-    <mapID target="au.gov.asd.tac.constellation.views.tableview.about" url="table-view-old.html"/>
```
Comment (antares1470) [inline CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/docs/tableview-map.xml:4]: similar

**Review by antares1470 — CHANGES_REQUESTED** [review]
If you're also able to go through some of the images in the docs resources and remove any images only used by the old table view help, that would be great.

If you're also able to add an entry to whatsnew.txt mentioning that this has been removed

**Comment by formalhaunt** [comment]
Ok. I am a little confused. But I removed the commit. Restored the files. Re-did the commit and pushed. I think because the commit was reversed and there were no changes the PR automatically closed itself. Re-opened and should be good.

I have restored the docs folder and moved it into the tableview2 package. I have renamed the package path in the files where I could find it. Hopefully its all working now.

**Review by antares1470 — CHANGES_REQUESTED** [review]
I've given this a test and seems to have removed successfully. A couple of things though:
- Are you able to add an entry to whatsnew.txt so that users are informed of the fact that the old table view has been removed?
- Are you possibly able to rename the tableview2 packages to just tableview? With the old table view gone, there isn't much reason to keep the package name as tableview2

**Comment by formalhaunt** [comment]
Added an entry into `whatsnew.txt`.

In regards to the package rename. Can we just hold off for a little bit. I am in the middle of a rather large refactor atm. Not sure if it will be merged but if it is this could cause me some conflict headaches.

**Comment by antares1470** [comment]
> In regards to the package rename. Can we just hold off for a little bit. I am in the middle of a rather large refactor atm. Not sure if it will be merged but if it is this could cause me some conflict headaches.

Sure that's ok. Definitely isn't something that can't wait

File: CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt [hunk 3]
```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2021-08-18 ==
```
Comment (antares1470) [inline CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt:4]: Are you able to add a title? Can just be something simple like "Table View"

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by arcturus2 — APPROVED** [review]

File: CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt (hunk 3 above)
Comment (formalhaunt) [reply CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt]: Added

**Review by antares1470 — APPROVED** [review]

**Pull request merged**